frontend/control-center/src/components/Sidebar/index.tsx

```diff
@@ -29,9 +28,8 @@ const Sidebar = (props: SideBarProps) => {
     return useMatch(`${route}/*`);
   };
 
-  const webhooksEnabled =
-    props.catalog[`${ComponentRepository.airyCore}/${ConfigServices.integrationWebhook}`]?.installed;
-  const inboxEnabled = props.components[ConfigServices.frontendInbox]?.enabled;
+  const webhooksEnabled = props.components[ComponentName.integrationWebhook]?.enabled || false;
+  const inboxEnabled = props.components[ComponentName.frontendInbox]?.enabled || false;
```

Comment:
```
 const webhooksEnabled =
    props.catalog[`${ComponentRepository.airyCore}/${ConfigServices.integrationWebhook}`]?.installed;
```

this line was correct because Webhooks need to be in the sidebar only when they are installed